.travis.yml

```diff
@@ -19,3 +19,5 @@ before_install:
   wget https://${fossalocation}/${fossafile} && tar xzf ${fossafile}
   && ./fossa init
   && GO111MODULE=on FOSSA_API_KEY=${fossapush} ./fossa analyze -t kubernetes-ingress -b ${TRAVIS_BRANCH}; fi
+- echo "Checking ingress controller for linting errors" && wget -O - -q ${golangci_lint} | sh -s v1.17.1
```

Comment:
`sh -s v1.17.1` ? 
judging by the output from here https://travis-ci.org/nginxinc/kubernetes-ingress/builds/575403783?utm_source=github_status&utm_medium=notification, `$ export golangci_lint_version=v1.17.1`,  the env variable was supposed to be used here

Reply:
I decided against using a travisci variable as you saw in that log there. 
Version of the linter should be tracked in version control.